script.js

```diff
@@ -1,7 +1,7 @@
 function calc() {
```

Comment:
You messed your git up a bit here. Remember that Pull Requests are per branch and branches don't end when you make one. All you had to do was make this change in your existing branch, commit that change and push that branch up and the existing pull request you had would have been updated accordingly.